install/kubernetes/helm/istio/values.yaml

```diff
@@ -157,7 +157,8 @@ ingressgateway:
     type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
     ports:
       ## You can add custom gateway ports
-    - port: 80
+    - port: 12345
```

Comment:
Why ? Are you sure this works ?

Reply:
Also we should stop making up random port number. Stick with the 150xx range.

Reply:
@costinm so how to stop making up random port number? As all of the ports here in values.yaml can be some random ports and customer can customize those port based on their requirement to avoid port conflict.

The reason that I need to have this PR is because I want to avoid service port conflict, please refer to https://github.com/istio/istio/issues/6000#issue-328886754 for detail of the issue.